### PR TITLE
Coloration

### PR DESCRIPTION
**Pull request by martha-thomae**
This pull request implements coloration by including the keystrokes to enter colored notes, which are translated into their corresponding tokens in **mens (recently included into Humdrum).

### CODE REVIEW

**Comment by karend27** [comment]
Hi @martha-thomae - was wondering if coloration has been merged in?

**Comment by martha-thomae** [comment]
Hi @karend27! Thanks for the reminder, not yet. @JRegimbal, would you mind merging this PR if you agree with the changes? thanks!

**Comment by JRegimbal** [comment]
Oops sorry I didn't realize this was waiting on me. It seems good from my end!

**Comment by martha-thomae** [comment]
Perfect, thank you! 
Can you merge, @JRegimbal ? I don't seem to have permissions for this.

**Comment by JRegimbal** [comment]
Oh I manually added it in already. I don't know why it isn't marked...let me try again to see if I can make github happy.

**Comment by martha-thomae** [comment]
Thanks! :)
